docs/user/security/rbac_tutorial.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[space-rbac-tutorial]]
+=== Tutorial:  Use Role Based Access Control (RBAC) to customize Kibana Spaces
+
+Role Based Access Control (RBAC) is used to provide access to data, tools,
+and Kibana Spaces.  In this tutorial you will learn how to configure roles
+that provide the right users with the right access to the data, tools, and
+Kibana Spaces.
+
+[float]
+==== Scenario
+
+Our user is a web developer working on a bank's
+online mortgage service.  The web developer has these 
+three requirements:
+
+* Have access to the data for that service 
+* Build visualizations and dashboards
+* Monitor the performance of the system
+
+You'll provide the web developer with the access and privileges to get the job done.
+
+[float]
+==== Prerequisites
+
+To complete this tutorial, you'll need the following:
+
+*  **Administrative privileges**: To create a space, role, and a user.  
```

Comment:
You must have a role that grants privileges to create a space, role, and user.  See the  {ref}/built-in-roles.html[built-in] roles.